src/REPLMode.jl

```diff
@@ -1104,7 +1104,10 @@ packages have changed causing the current Manifest to_indices be out of sync.
     ["activate"],
     do_activate!,
     (ARG_RAW, [0,1]),
-    [],
+    [
+        ("local", OPT_SWITCH, :envdir => true),
+        ("shared", OPT_SWITCH, :envdir => false),
```

Comment:
I think the kwargs here are better as `:local => true` and `:local => false`.

Reply:
I don't think I can use `local` because its is a keyword that I can't use as a kw argument idnetifier. I'll try `shared` instead